modules/ROOT/pages/getting-started-libvirt.adoc

```diff
@@ -24,14 +24,12 @@ VCPUS="2"
 RAM_MB="2048"
 DISK_GB="10"
 
-virt-install --connect="qemu:///system" --name="${VM_NAME}" --vcpus="${VCPUS}" --memory="${RAM_MB}" \
-        --os-variant="fedora32" --import --graphics="none" \
+virt-install --connect=qemu:///system --name="${VM_NAME}" --vcpus=${VCPUS} --memory=${RAM_MB} \
+        --os-variant=fedora-coreos-$STREAM --import --graphics=none \
```

Comment:
Let's not remove the quotes at lest for entries that have variables since the others have them:

```suggestion
virt-install --connect=qemu:///system --name="${VM_NAME}" --vcpus="${VCPUS}" --memory="${RAM_MB}" \
        --os-variant="fedora-coreos-${STREAM}" --import --graphics=none \
```